StepperEngine/src/Flow/FlowRunHistory.java

```diff
@@ -0,0 +1,89 @@
+package Flow;
+
+import DataTypes.DataType;
+import Steps.Step;
+
+public class FlowRunHistory {
+    private String flowId;
+    private String flowName;
+    private Flow.Status status;
+    private double runTime;
+    private String timeStamp;
+
+    private String freeInputsHistory;
+    private String outputsHistory;
+    private String stepsHistory;
```

Comment:
I think these are fine for now, but maybe not for the next assignment with the GUI - we might need to change it then.

for example - what if the GUI needs to present "freeInputHistory" not as a whole string, but divided into different inputs (and maybe different fields of each input) in different GUI components (i.e text boxes)

Reply:
you are right, i created them that way so we could progress faster in the current task.
in the following day I will add fields and methods, so FlowRunHistory will be more suited to further tasks